src/main/org/apache/tools/ant/util/jarattr/Constant.java

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
There is another implementation in `org.apache.tools.ant.taskdefs.optional.depend.constantpool`; perhaps that could be reused?